src/include/meta.h

```diff
@@ -33,11 +33,16 @@
 #define WT_HS_FILE "WiredTigerHS.wt"     /* History store table */
 #define WT_HS_URI "file:WiredTigerHS.wt" /* History store table URI */
 
-#define WT_SYSTEM_PREFIX "system:"               /* System URI prefix */
-#define WT_SYSTEM_CKPT_TS "checkpoint_timestamp" /* Checkpoint timestamp name */
-#define WT_SYSTEM_CKPT_URI "system:checkpoint"   /* Checkpoint timestamp URI */
-#define WT_SYSTEM_OLDEST_TS "oldest_timestamp"   /* Oldest timestamp name */
-#define WT_SYSTEM_OLDEST_URI "system:oldest"     /* Oldest timestamp URI */
+#define WT_SYSTEM_PREFIX "system:"                               /* System URI prefix */
+#define WT_SYSTEM_CKPT_TS "checkpoint_timestamp"                 /* Checkpoint timestamp name */
+#define WT_SYSTEM_CKPT_URI "system:checkpoint"                   /* Checkpoint timestamp URI */
+#define WT_SYSTEM_OLDEST_TS "oldest_timestamp"                   /* Oldest timestamp name */
+#define WT_SYSTEM_OLDEST_URI "system:oldest"                     /* Oldest timestamp URI */
+#define WT_SYSTEM_CKPT_SNAPSHOT "snapshots"                      /* List of snapshots */
```

Comment:
Could this be treated as a list in the configuration parser, rather than having a special serialized format?

Reply:
Thanks, @agorrod. Updated the code to use list.